install_config/upgrading/manual_upgrades.adoc

```diff
@@ -215,7 +280,6 @@ endif::[]
 +
 ----
 # oadm policy reconcile-cluster-role-bindings \
-    --exclude-groups=system:authenticated \
     --exclude-groups=system:authenticated:oauth \
```

Comment:
Ah OK, thanks. Add it to both `oadm policy reconcile-cluster-roles` commands above, or just the 2nd? We have two invocations in the procedure currently.